src/features/latest-news/redux/actions.ts

```diff
@@ -4,7 +4,7 @@ import { ActionWithPayload } from "store/actions";
 export type LatestNewsData = {
   headline: string;
   source: string;
-  date: string;
+  date: Date;
```

Comment:
I'm still skeptical if the server is actually sending a `Date` format to you. The API sends a number (e.g. `1624088851026`) which isn't a `Date` format, so maybe this type should be of `number`

Reply:
I agree.

Reply:
If it's sending a "epoch time" (milliseconds since 1st January, 1970), then it should be a `number`.
If it's sending an "ISO-8601" value (`YYYY-MM-DDTHH:mm:ss±hhmm`), then it should be a `string`.

However, this could validly be a `Date` here, if you're converting it from `number`/`string` to `Date` in your middleware before dispatching the data into redux.